src/scales/quantitativeScale.ts

```diff
@@ -42,6 +51,44 @@ module Plottable {
       return super.domain(values); // need to override type sig to enable method chaining:/
     }
 
+    /**
+     * Gets the minimum domain value when autoDomain()-ing. 
+     */
+    public min(): D;
+    /**
+     * Sets the minimum domain value when autoDomain()-ing.
+     * 
+     * @returns {QuantitativeScale<D>} The calling QuantitativeScale.
+     */
+    public min(value: D): QuantitativeScale<D>;
+    public min(value?: D): any {
+      if (value === undefined) {
```

Comment:
This is so weird as it does not happen anywhere else in our codebase. I suggest adding a third method called `removeMinMaxConstraints()` or something which removes constraints for both min and max